### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by stevenhalverson**

### CODE REVIEW

File: get_download_test.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from selenium import webdriver
```
Comment (eddyizm) [inline get_download_test.py:1]: Let's try to change this using an internal library like `requests` - I've had several issues running selenium on a webserver where this is going to ultimately run.

Comment (stevenhalverson) [reply get_download_test.py:1]: Gotcha. Will find a way to do it without selenium.

Comment (stevenhalverson) [reply get_download_test.py]: Good news is it looks like the URL changes on a consistent pattern (every second Monday of the month) so `requests` should be able to work for this.

Comment (stevenhalverson) [reply get_download_test.py]: Did you want this program constantly running in the program or something you use intermittently?

Comment (eddyizm) [reply get_download_test.py]: This is something that will be run weekly or monthly. This is a great start that can get fleshed out. We will worry about the schedule and validation later.

**Review by eddyizm — APPROVED** [review]
Good work. We will improve on this as it gets implemented we work out the rest of the logic.

**Comment by stevenhalverson** [comment]
Thanks. My skills are pretty limited so if there's anything else I can help with just let me know.

**Pull request merged**